### ISSUE TITLE
Add Sending Patches / Banks to the DW-8000

### ISSUE THREAD

**Issue by planist79** [christofmuc/KnobKraft-orm#295]
Hello,

i would like to request "Add Sending Patches / Banks to the DW-8000"
It says in the log window of version 2.1 that "Banks to the DW-8000" is not implmented yet.

Is this possible to do? Or could i do according to some other example for synths that are already supported?

Thank you!!!

**Comment by planist79** [comment]
![Uploading Knobkraft DW-8000.jpg…]()

**Comment by christofmuc** [comment]
I reopened #177, where I had proclaimed a year ago that it was possible. Looking at my text now, I probably think I wanted to say it would be possible to implement with the 2.0.0, not that I had done it, but then who knows. 

We have a beta version of the DW-8000 adaptation. If that works, we can decide whether to extend the C++ code for the next release or just ship the Python code. I think the only difference is that the C++ version also displays parameter values in the metadata section and the patch comparison.
